test/controllers/tasklist/VariationProgressSpec.scala

```diff
@@ -313,7 +317,8 @@ class VariationProgressSpec extends SpecBase {
 
               result.mandatory mustBe List(
                 Task(Link(TrustDetails, s"http://localhost:9838/maintain-a-trust/trust-details/$identifier"), Completed),
-                Task(Link(Assets, s"http://localhost:9800/maintain-a-trust/trust-assets/$identifier"), NotStarted)
+                Task(Link(Assets, s"http://localhost:9800/maintain-a-trust/trust-assets/$identifier"), NotStarted),
+                Task(Link(TaxLiability, s"http://localhost:9844/maintain-a-trust/tax-liability/$identifier"), NoActionNeeded)
```

Comment:
@jackheslop96 I don't think this would be quite right as if there is no action needed we want "Tax liability" text but for it to not be a hyperlink.